packages/harmony/svgr-template.js

```diff
@@ -0,0 +1,33 @@
+const template = (variables, context) => {
+  const { tpl } = context
+
+  variables.jsx.openingElement.name.name = 'Icon'
+  variables.jsx.closingElement.name.name = 'Icon'
+
+  // Append a {...props} to the opening element's attributes
+  variables.jsx.openingElement.attributes.push({
+    type: 'JSXSpreadAttribute',
+    argument: {
+      type: 'Identifier',
+      name: 'props'
+    }
+  })
```

Comment:
wut how does this work

Reply:
Had to dig deep into the babel docs... 😰

Reply:
If adding comments to this file doesn't break the transformation (or cause the comments to be part of the output), it might be a good idea to add them in here explaining how it works.
Or maybe just a readme next to it?

I _think_ I understand it from reading the linked docs... 😅